rfcs/text/0001-rfc-process.md

```diff
@@ -70,103 +84,111 @@ Where necessary, existing RFCs can be revised using the same process (a proposed
 
 ## Detailed Design
 
-### Proposing RFCs
+### Drafting RFCs
 
 #### **Authors**:
 
-*Before proposing, always engage the DCP community to build consensus and assess whether your proposal sparks interest and addresses a need. You may even discover potential collaborators.*
+There are three types of RFCs that may be proposed: ***Software***, ***Governance***, and ***Informational***. Software RFCs are expected to be linked to an Objective that is prioritized in the DCP Product Roadmap, tracked by linking directly to an existing ticket in the DCP Zenhub board (usually labeled as a Spike). Informational and Governance RFCs need not directly reference an open ticket in the DCP Zenhub board.
 
-  - Fork the HumanCellAtlas dcp-community repository
-  - Copy `rfcs/rfc-template.md` to `rfcs/text/0000-my-feature.md`
-  *- where "my-feature" is descriptive. Don't assign an RFC number yet.*
-  
-  - Fill in the RFC with attention to detail
+To inform the DCP Community about the development of an RFC, with the intention of forming collaborations between people thinking about similar problems, it is RECOMMENDED that a draft of the RFC be published as soon as the Problem Statement and Authors have been established. The steps to begin are as follows:
 
-  - Submit a pull request.
-  
-      For software RFCs:
-      - Add the "rfc-community-review" and the appropriate software project name _(such as "Data Store")_ labels. When the RFC impacts multiple DCP software projects, then the "Architecture" label **MUST** be the project name. *If uncertain about the appropriate project name, then **_Ask a PM_** on the HCA **#dcp-project-mgmt** slack channel*
+   1. Fork the HumanCellAtlas `dcp-community` repository
+   2. Copy `rfcs/rfc-template.md` to `rfcs/text/0000-my-feature.md`. Make `my-feature` descriptive but do not assign an RFC number yet.
+   3. Fill out the Problem Statement and Author(s) sections.
+   4. Push the changes to your branch and create a _Draft_ PR.
+   5. Label the PR with the "rfc-draft" label.
+
+#### **Non-Authors**:
+
+When a _Draft_ RFC PR is published and labeled with "rfc-draft," specific commentary is NOT RECOMMENDED (negative or positive) until the Authors(s) propose the RFC for community review. If the draft RFC sparks interest, it is RECOMMENDED that one reach out to the Author(s) offline to suggest a collaboration.
 
-      For governance RFCs:
-        
-      - Add the "rfc-community-review" and the "Governance" labels. 
+### Proposing RFCs
+
+#### **Authors**:
 
-- Add a minimum two week _last call_ for the completion of the Community review to the top-level summary comment in the RFC pull request.
+  - Fill in the RFC with attention to detail
+  
+  - Submit a pull request. If the PR was originally in a draft state, remove the "rfc-draft" label.
+  
+  - Add the "rfc-community-review" label.
+  
+  - Find the auto-generated ticket in the Zenhub `dcp-community` board and assign all Author(s) to the ticket.
```

Comment:
It's not really an _auto-generated ticket_ - the github pull request is just visible on the related board for the repo. In this case, it's not different from assigning an owner to the issue in the repo. Is there an intent to define a specific set of pipelines in zenhub for modeling RFC process similar to [Modeling progress with ZenHub Pipelines [component repository]](https://github.com/HumanCellAtlas/dcp-community/blob/master/rfcs/text/0012-roadmaps%2Bplanning.md#modeling-progress-with-zenhub-pipelines-component-repository).